src/App.jsx

```diff
@@ -25,6 +25,7 @@ function App () {
         <button onClick={() => setCount(count + 1)}>+</button>
       </span>
       <div>Hello! This is Madaleine & Hunter.</div>
+      <p>Hey Hunter, you should sell porcupine pets for a living.</p>
```

Comment:
What if I sell them with hotdogs on the spines?? I feel like that will supplement my income enough to lessen my porcupine use?